modules/ztp-configuring-hwevents-using-pgt.adoc

```diff
@@ -36,34 +31,95 @@ You can configure bare-metal hardware events for vRAN clusters that are deployed
   policyName: "subscriptions-policy"
 ----
 
-. Add the `Interconnect` CR to `.spec.sourceFiles` in the site configuration file, for example, the `example-sno-site.yaml` file:
+. Add the `HardwareEvent` CR to `spec.sourceFiles` in your specific group configuration file, for example, in the `group-du-sno-ranGen.yaml` file:
 +
 [source,yaml]
 ----
-- fileName: AmqInstance.yaml
+- fileName: HardwareEvent.yaml <1>
   policyName: "config-policy"
+  spec:
+    nodeSelector: {}
+    storageType: <storage_class_name> <2>
+    logLevel: "info"
```

Comment:
The source cr [HardwareEvent.yaml](https://github.com/openshift-kni/cnf-features-deploy/blob/release-4.13/ztp/source-crs/HardwareEvent.yaml) also sets amq as transportHost.

To configure http transport, the PGT has to overwrite the source cr transportHost :
spec:
    nodeSelector: {}
    storageType: <storage_class_name> 
   transportHost: "http://hw-event-publisher-service.openshift-bare-metal-events.svc.cluster.local:9043"
    logLevel: "info"